src/blog/2023/09/modernize-your-legacy-industrial-data.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Modernize your legacy industrial data
+subtitle: Working with legacy industrial protocol data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+description: Working with legacy industrial protocol data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
```

Comment:
I'll disagree and commit based on results value: https://flowfuse.com/handbook/company/values/#%F0%9F%93%88-results, but I do think this is too high level for the content being fairly low-level. This discrepancy will impact how useful it is for customers.